### PR TITLE
Add workflow for building on ubuntu

### PR DESCRIPTION
**Pull request by danada**
Circle CI reporting is weird and I think github actions are cleaner so I'd like to move away from circle CI and just use GH Actions for now. There are some issues in getting other OSes going, but I don't think we lose anything moving away from circle CI.

### CODE REVIEW

File: .github/workflows/continuous-build.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# MIT License
+#
+# Copyright (c) 2019 Choko (choko@curioswitch.org)
```
Comment (hoshir) [inline .github/workflows/continuous-build.yml]: Is this file copied from Curiostack? If so, the current copyright statement is right. 
If you wrote this file by yourself, the copyright should be Infostellar.

Comment (danada) [reply .github/workflows/continuous-build.yml]: It's copied from Curiostack in its entirety.

Comment (hoshir) [reply .github/workflows/continuous-build.yml]: That's fine in that case. Thanks.

**Comment by danada** [comment]
Windows and mac builds are failing due to some conan stuff. You can see the broken builds here: https://github.com/danada/stellarstation-api/actions?query=workflow%3A%22Continuous+Build%22

Will continue looking into this.

**Comment by danada** [comment]
Mac build is blocked by #232

**Comment by danada** [comment]
Switched to just ubuntu for now.

**Pull request merged**